packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -126,13 +166,35 @@ export default class QuanticSearchBox extends LightningElement {
     }
     this.state = this.searchBox?.state;
     this.suggestions =
-      this.state?.suggestions?.map((s, index) => ({
+      this.state?.suggestions?.map((suggestion, index) => ({
         key: index,
-        rawValue: s.rawValue,
-        value: s.highlightedValue,
+        rawValue: suggestion.rawValue,
+        value: suggestion.highlightedValue,
       })) ?? [];
   }
 
+  updateRecentQueriesListState() {
+    if (this.recentQueriesList.state?.queries) {
+      this.recentQueries = this.recentQueriesList.state.queries;
+      setItemInLocalStorage(
+        this.localStorageKey,
+        this.recentQueriesList.state.queries
+      );
+    }
+  }
+
+  /**
+   * Clears the recent queries.
+   */
+  clearRecentQueries = (event) => {
+    event.stopPropagation();
+    this.recentQueriesList.clear();
+  };
+
+  get localStorageKey() {
+    return `${this.engineId}_quantic-recent-queries`;
+  }
```

Comment:
we can of course but I don't see what this would add comparing to a getter?
for me they are exactly the same with one small advantage  for the  getter which is that it is more easy to notice when reading the code.

Reply:
A getter is code executed every time the property is called, so everytime you do `this.localStorageKey` you are creating a function, executing it, evaluating `this.engineId`, etc. to return a string that won't change after initialization.

It's a question of efficiency of code AFAIK.

Reply:
Good point 👍